src/RelationViewer/RelationViewerProvider.ts

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as vscode from 'vscode';
+
+import {getRelationData} from './example/RelationExample';
+import {RelationViewer} from './RelationViewer';
+
+export class RelationViewerDocument implements vscode.CustomDocument {
+  private readonly _uri: vscode.Uri;
+  private _metadataViwer: RelationViewer[];
```

Comment:
```suggestion
  private _relationViewer: RelationViewer[];
```

Maybe a typo?

Reply:
Oh, I'm sorry. I'll apply it right away.